### PR TITLE
Spar 29: Adding support for querying multiple collections

### PR DESCRIPTION
**Pull request by mehtakash93**
The test has been added to TestQuerying.scala. The collections that we are querying to must have the same schema

### CODE REVIEW

File: src/main/scala/com/lucidworks/spark/util/SolrSupport.scala [hunk 1]
```diff
@@ -449,19 +449,20 @@ object SolrSupport extends Logging {
...
-      collections = Array(collection)
-    }
-    else {
-      val aliases: Aliases = zkStateReader.getAliases
-      val aliasedCollections: String = aliases.getCollectionAlias(collection)
-      if (aliasedCollections == null) {
-        throw new IllegalArgumentException("Collection " + collection + " not found!")
-      }
-      collections = aliasedCollections.split(",")
+for(col <- collection.split(",")) {
```
Comment (thelabdude) [inline src/main/scala/com/lucidworks/spark/util/SolrSupport.scala]: fix the indentation here to match the surrounding code

File: src/main/scala/com/lucidworks/spark/util/SolrSupport.scala [hunk 2]
```diff
@@ -449,19 +449,19 @@ object SolrSupport extends Logging {
     val zkStateReader: ZkStateReader = solrClient.getZkStateReader
     val clusterState: ClusterState = zkStateReader.getClusterState
     var collections = Array.empty[String]
-
-    if (clusterState.hasCollection(collection)) {
-      collections = Array(collection)
+  for(col <- collection.split(",")) {
```
Comment (kiranchitturi) [inline src/main/scala/com/lucidworks/spark/util/SolrSupport.scala]: The indentation is still not right. Make sure the indentation aligns with the code above

File: src/main/scala/solr/DefaultSource.scala [hunk 3]
```diff
@@ -9,7 +9,7 @@ class DefaultSource extends RelationProvider with CreatableRelationProvider with
 
   override def createRelation(sqlContext: SQLContext, parameters: Map[String, String]): BaseRelation = {
     try {
-      new SolrRelation(parameters, sqlContext)
+      return new SolrRelation(parameters, sqlContext)
```
Comment (kiranchitturi) [inline src/main/scala/solr/DefaultSource.scala]: no need of explicit return

**Comment by kiranchitturi** [comment]
Looks good. Commented on the minor changes.
All tests are passing https://jenkins.lucidworks.com/view/spark-solr/job/spark-solr-oneoff/73/

**Comment by kiranchitturi** [comment]
buildme

**Comment by kiranchitturi** [comment]
buildme

**Pull request merged**